apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import { cloneDeep } from 'lodash';
```

Comment:
using lodash is not recommended, why not using the native solution I suggested ?